### PR TITLE
Update hot-module-replacement.md

### PR DESCRIPTION
**Pull request by kryptokinght**
Simplifying the language

### CODE REVIEW

File: src/content/concepts/hot-module-replacement.md [hunk 1]
```diff
@@ -13,7 +14,7 @@ Hot Module Replacement (HMR) exchanges, adds, or removes [modules](/concepts/mod
 
 - Retain application state which is lost during a full reload.
 - Save valuable development time by only updating what's changed.
-- Tweak styling faster -- almost comparable to changing styles in the browser's debugger.
+- Live CSS modification -- almost comparable to changing CSS in the browser's debugger.
```
Comment (Legends) [inline src/content/concepts/hot-module-replacement.md]: What about?:

> CSS and JS source code modifications result in instant view/browser updates -- almost comparable to changing CSS/JS in the browser's dev tools directly.

Comment (kryptokinght) [reply src/content/concepts/hot-module-replacement.md]: Ahh, that's better. Will apply the changes to the files and commit it again.

**Comment by kryptokinght** [comment]
@Legends updated according to your suggestion

**Comment by kryptokinght** [comment]
@montogeek please review this PR

File: src/content/concepts/hot-module-replacement.md [hunk 2]
```diff
@@ -13,7 +14,7 @@ Hot Module Replacement (HMR) exchanges, adds, or removes [modules](/concepts/mod
 
 - Retain application state which is lost during a full reload.
 - Save valuable development time by only updating what's changed.
-- Tweak styling faster -- almost comparable to changing styles in the browser's debugger.
+- Modifications made to CSS/JS source code in your IDE result in instant view/browser updates -- almost comparable to changing CSS/JS in   the browser's dev tools directly.
```
Comment (EugeneHlushko) [inline src/content/concepts/hot-module-replacement.md]: Modifications made to CSS/JS in the source code results in an instant browser update which is almost comparable to changing styles directly in the browser's dev tools.

**Comment by kryptokinght** [comment]
@EugeneHlushko updated accordin to your suggestion

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**